docs/src/pages.js

```diff
@@ -161,7 +161,19 @@ const pages = [
     pathname: '/api-docs',
     children: [
       ...findPages[0].children,
-      ...[{ pathname: '/api-docs/data-grid' }, { pathname: '/api-docs/x-grid' }],
+      {
+        pathname: '/api-docs/data-grid',
+        title: 'Data Grid',
+        children: [
+          { pathname: '/api-docs/data-grid', title: 'API Reference' },
```

Comment:
On the landing page it's a sub-item (https://deploy-preview-26477--material-ui.netlify.app/) but once you go to https://deploy-preview-26477--material-ui.netlify.app/api/data-grid/, the nested menu structure disappears.

Reply:
Because the redirect rule sends to https://master--material-ui-x.netlify.app/api/data-grid/. Once mui-org/material-ui-x#1529 is merged it will work.